src/utilities/routerNavigation.browser.spec.ts

```diff
@@ -16,59 +15,35 @@ test('when go is called, forwards call to window history', () => {
 })
 
 test('when back is called, forwards call to window history', () => {
-  vi.spyOn(window.history, 'back')
+  vi.spyOn(window.history, 'go')
 
   const onAfterLocationUpdate = vi.fn()
   const history = createRouterNavigation({ onAfterLocationUpdate })
 
   history.back()
 
-  expect(window.history.back).toHaveBeenCalledOnce()
+  expect(window.history.go).toHaveBeenCalledOnce()
 })
 
 test('when forward is called, forwards call to window history', () => {
-  vi.spyOn(window.history, 'forward')
+  vi.spyOn(window.history, 'go')
 
   const onAfterLocationUpdate = vi.fn()
   const history = createRouterNavigation({ onAfterLocationUpdate })
 
   history.forward()
 
-  expect(window.history.forward).toHaveBeenCalledOnce()
-})
-
-test('when update is called, calls updateBrowserUrl', () => {
-  vi.spyOn(utilities, 'updateBrowserUrl')
-
-  const onAfterLocationUpdate = vi.fn()
-  const url = random.number().toString()
-  const history = createRouterNavigation({ onAfterLocationUpdate })
-
-  history.update(url)
-
-  expect(utilities.updateBrowserUrl).toHaveBeenCalledWith(url, undefined)
```

Comment:
Not testing, because we're not responsible. But we could probably mock the history library and test that the right history method is called.

Reply:
yep, makes sense to me